### ISSUE TITLE
Mapping missing roads in Manila

### ISSUE THREAD

**Issue by GRABOSM** [GRABOSM/Grab-Data#33]
**Objective:** Continuing from our [previous efforts](https://github.com/GRABOSM/Grab-Data/issues/19), our data team will be going through the existing road geometries and add the missing ones if any. We'll be reviewing and mapping the following features: missing roads, road classification, and check for intersection configurations and alignment accuracy.

![Pic_geo_photos_-_ph=mm=muntinlupa=slex_-_view_from_bilibid_overpass_-philippines--2015-0428--ls-](https://user-images.githubusercontent.com/38514914/61107267-27704280-a49d-11e9-836a-803f22ffdb01.jpg)
_By Patrickroque01 at English Wikipedia, CC BY-SA 3.0, https://commons.wikimedia.org/w/index.php?curid=54468580_

**Cities/region we are reviewing/mapping:** 
	
- Manila
(Will update the list of cities as we go ahead)

**Workflow:** We'll be following the same workflow as mentioned in our wiki - https://github.com/GRABOSM/Grab-Data/blob/master/Data%20Improvement%20Projects and https://wiki.openstreetmap.org/wiki/Grab#Process_Flow

**Validation:** We'll be following the validation guidelines mentioned in our wiki: https://wiki.openstreetmap.org/wiki/Grab#Quality_Process

**Changeset Comment**: `Mapping missing roads and road attributes https://github.com/GRABOSM/Grab-Data/issues/33 #Grab`

**Source:** Digital Globe Standard, Bing Aerial Imagery, ESRI World Imagery, and Maxar for freshness 

**Filter:** None

**Plugins:** None

**Edge cases:** None at the moment, we'll post it as a subsequent comment if need be.

**Team involved:** [The Grab Data team](https://github.com/GRABOSM/Grab-Data/blob/master/Grab%20Data%20Team)

Do reach out to us if you have any questions or suggestions on the same. Happy Mapping! 

cc: @jinalfoflia

**Comment by maning** [comment]
I posted this in OSM PH telegram channel, posting here for posterirty and visvibility to the whole grab mapping group.  I live in this area and verifies all changes in this road before editing.

<img width="459" alt="Screenshot 2019-08-26 15 34 59" src="https://user-images.githubusercontent.com/353700/63673532-13975a80-c817-11e9-9b03-3eafcd5b18e5.png">

**Comment by govvin** [comment]
![image](https://user-images.githubusercontent.com/5821688/63823001-d51fae00-c984-11e9-9130-ab2d589ed292.png)

Why would your mappers add a motor_vehicle access tag to a ferry route?

Also, the local community in Metro Manila has inquired with the local map ops team in the past, but we never got any specific reply about any quality assurance process that your local team has adopted. We are really concerned with the inconsistent quality of edits your team has been adding to the OSM database.

**Comment by jinalfoflia** [comment]
Thank you for flagging these edits, @maning. We'll make sure to be more aware of it and comment on the changesets when in doubt. 

@govvin:

>Why would your mappers add a motor_vehicle access tag to a ferry route?

Thank you for flagging this, our team has cross-checked and it was an error from our side. We'll be more careful about it here onwards. 

>Also, the local community in Metro Manila has inquired with the local map ops team in the past, but we never got any specific reply about any quality assurance process that your local team has adopted. We are really concerned with the inconsistent quality of edits your team has been adding to the OSM database.

We understand this concern and we are closely working with the local map operations team in teaching, learning and always improving the process of understanding mapping in OpenStreetMap. We really appreciate the feedback we get from the community and we'll make sure we iterate our quality check process based on this. Thank you so much!

**Comment by ockim16** [comment]
Good day! 

As per discussion in the LoCos Telegram channel, we have reverted the changes done under the Pasig City Ferry Route using http://revert.osmz.ru/2399. All changes have been reverted under changeset  74561008 (https://www.openstreetmap.org/changeset/74561008). Thank you and again apologies.

**Comment by maning** [comment]
Thank you @ockim16.

This ticket has [over 1,300  changesets](https://osmcha.mapbox.com/?filters=%7B%22date__gte%22%3A%5B%7B%22label%22%3A%22%22%2C%22value%22%3A%22%22%7D%5D%2C%22comment%22%3A%5B%7B%22label%22%3A%22https%3A%2F%2Fgithub.com%2FGRABOSM%2FGrab-Data%2Fissues%2F33%22%2C%22value%22%3A%22https%3A%2F%2Fgithub.com%2FGRABOSM%2FGrab-Data%2Fissues%2F33%22%7D%5D%7D) associated with it.  **Please check them and confirm if edits are valid.**

I my first pass, some of them are not even in the Philippines.

**Comment by GRABOSM** [comment]
>This ticket has over 1,300 changesets associated with it. Please check them and confirm if edits are valid.

@maning our team has reviewed these edits and we have made the required changes. Thank you for flagging these issues, it helped us to strengthen our processes. Thank you for helping us understand the rationale behind these issues.


>In my first pass, some of them are not even in the Philippines.

This is an error from our team's side as they continued using the same changeset comment even though they moved to a different region. We have commented the right changeset on the impacted ones, for instance: https://www.openstreetmap.org/changeset/72805070

cc: @jinalfoflia
